lms/security.py

```diff
@@ -0,0 +1,176 @@
+import base64
```

Comment:
This is a relativly large file but I think still make sense to keep all here unless it keeps growing, we could later split int lms/security/policies etc